ui/nuxeo-filter.js

```diff
@@ -183,12 +184,7 @@ import Interpreter from './js-interpreter/interpreter.js';
       let res = false;
 
       try {
-        if (
-          Nuxeo.UI &&
-          Nuxeo.UI.config &&
-          Nuxeo.UI.config.expressions &&
-          String(Nuxeo.UI.config.expressions.eval) === 'false'
-        ) {
+        if (String(config.get('Nuxeo.UI.config.expressions.eval')) === 'false') {
```

Comment:
This is related with what @nelsonsilva discuss in `config.js` file, right? If we could have booleans, would make this much cleaner.

Reply:
The problem is that the configuration string always returns a string or a list of strings. How do we know if "false" should be converted to a boolean or read as a string?

Reply:
🤔 true, not easy to assert that

Reply:
`config.get('Nuxeo.UI.config.expressions.eval', false)` tells the `get` method that the property is meant to be a boolean. This should allow us to then, if the value is not of type boolean, to do the proper conversion, no?

Reply:
nevermind .. outdated but not closed I guess...

Reply:
Yes, I have already implemented this for primitive types. What about objects? Should we do a JSON.parse(JSON.stringify(...)) as @richardsd suggested to protected from unwanted changes?